elide-core/src/main/java/com/yahoo/elide/parsers/state/RecordTerminalState.java

```diff
@@ -85,12 +84,8 @@ public RecordTerminalState(PersistentResource record, CollectionTerminalState co
 
     @Override
     public Supplier<Pair<Integer, JsonNode>> handleDelete(StateContext state) {
-        try {
-            record.deleteResource();
-            return () -> Pair.of(HttpStatus.SC_NO_CONTENT, null);
-        } catch (ForbiddenAccessException e) {
-            return () -> Pair.of(e.getStatus(), null);
-        }
+        record.deleteResource();
```

Comment:
The only test that isn't passing is a JSON Patch extension, unfortunately :( i think it may be best to clean up that solution.

I'm relatively certain the reason it was this way is because it was easier for us to do at the time